### PR TITLE
add VkBadge

### PR DESCRIPTION
**Pull request by ameli90**

### CODE REVIEW

File: src/docs/pages/Badge/BudgeDefault.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+</template>
+
+<script>
+import * as Helper from '../../helper'
+import mixins from '../../mixins'
+
+export default {
+  mixins: [mixins],
+  data: () => ({
+    props: Helper.getProps('Budge', props),
```
Comment (miljan-aleksic) [inline src/docs/pages/Badge/BudgeDefault.vue]: fix typo

File: src/vue/Budge.vue [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export default {
+  props: {
+    style: {
+      type: String,
+      default: ''
+    }
+  },
+  computed: {
+    classes: function () {
+      return {
```
Comment (miljan-aleksic) [inline src/vue/Budge.vue]: simple classes as this one, keep them in the template it self

File: src/docs/pages/Badge/BudgeDefault.vue [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <docs-page
+            :props="props"
```
Comment (miljan-aleksic) [inline src/docs/pages/Badge/BudgeDefault.vue]: check identaton, only 2 spaces allowed

File: src/docs/pages/Badge/BudgeDefault.vue [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            :code="code">
+      <div slot="demo">
+        <vk-budge :style="props.style.value">
```
Comment (miljan-aleksic) [inline src/docs/pages/Badge/BudgeDefault.vue]: ref:demo is missing, here

File: src/docs/Docs.vue [hunk 5]
```diff
@@ -34,15 +34,17 @@ const Pages = {
...
 }
 
 export default {
   components: {
     DocButton: Pages.Button,
     DocModal: Pages.Modal,
     DocPagination: Pages.Pagination,
-    DocTabs: Pages.Tabs
+    DocTabs: Pages.Tabs,
+    DocBadge: Pages.Badge
```
Comment (miljan-aleksic) [inline src/docs/Docs.vue]: rebaste latest changes from master and change to PageBadge

File: src/docs/pages/Badge/BadgeDefault.vue [hunk 6]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <docs-page 
+      :props="props"
+      :code="code">
+      <div slot="demo">
+        <vk-badge :style="props.style.value" v-ref:demo>
```
Comment (miljan-aleksic) [inline src/docs/pages/Badge/BadgeDefault.vue]: Keep the format as in other docs:

```html
<vk-badge v-ref:demo
  :style="props.style.value">
...
```

File: src/vue/Badge.vue [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  props: {
+    style: {
+      type: String,
+      default: ''
+    }
+  },
+  computed: {
+    classes: function () {
+      return {
+        'uk-badge-notification': this.style === 'notification',
```
Comment (miljan-aleksic) [inline src/vue/Badge.vue]: Let's use `notice` instead of `notification`, its shorter and means the same.

Comment (miljan-aleksic) [inline src/vue/Badge.vue]: And don't forget to move them to the template ;)

**Comment by ameli90** [comment]
udpated

File: src/docs/pages/Badge/BadgeDefault.vue [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+export default {
+  mixins: [mixins],
+  data: () => ({
+    props: Helper.getProps('Badge', props),
+    code
+  })
+}
+
+const code = '<vk-badge style="notification">Information</vk-badge>'
```
Comment (miljan-aleksic) [inline src/docs/pages/Badge/BadgeDefault.vue]: For the code example keep all options by default and simple, like `<vk-badge></vk-badge>`

File: src/docs/pages/Badge/BadgeDefault.vue [hunk 9]
```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+    <docs-page 
+      :props="props"
+      :code="code">
+      <div slot="demo">
+        <vk-badge v-ref:demo
+          :style="props.style.value" >
```
Comment (miljan-aleksic) [inline src/docs/pages/Badge/BadgeDefault.vue]: remove the extra space

Comment (ameli90) [inline src/docs/pages/Badge/BadgeDefault.vue]: updated

**Pull request merged**